src/code/rand.c

```diff
@@ -88,12 +98,22 @@ void Rand_Seed(u32 seed) {
  * @note Original name: fqrand
  */
 f32 Rand_ZeroOne(void) {
+#if RAND_VERSION == RAND_N64
+    fu v;
+    f32 vf;
+
+    v.i = (Rand_Next() & 0x007FFFFF) | 0x3F800000;
+    vf = v.f - 1.0f;
```

Comment:
@Thar0 does your above comment

>  `@note` This technique for generating pseudo-random floats is recommended as a particularly fast but potentially
> non-portable generator in "Numerical Recipes in C: The Art of Scientic Computing", pp. 284-5.

still apply to this?

Reply:
Yes, it refers especially to the part where it's converting the random integer to a random float which is the same between both versions